### PR TITLE
HTML Signup Form with animation

### PR DESCRIPTION
**Pull request by danielufeli**
In this project we created a signup page in which:

- We developed the input fields
- We added CSS Styles and Animation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+      <li class="item6">turbotax</li>
+      </ul>
+        
+    </div>
+
+    <div class="form-container">
+      <form action="#" id="sigup-form">
+        <h2>Create an Intuit account</h2>
+        <p>One account for everything Intuit, including Mint. Learn more</p>
```
Comment (elmejdki) [inline index.html]: the learn more text should be wrapped between a link tag cause it should be a link not text.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          <label for="email">Email address</label>
+        </div>
+        <div>
+          <input type="email" name="email" id="email" />
+        </div>
+        <div>
+          <label for="phone">Phone (recommended)</label>
+        </div>
+        <div>
+          <input type="text" name="phone" id="phone" />
```
Comment (elmejdki) [inline index.html]: this input should be of type tel please change it.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job so far guys :clap: :+1: :100: , but in order to make your project approved you need to make some changes :hammer: :wrench: :toolbox: 

- [x] you have some linter errors please fix them:
image

- **[OPTIONAL]** you could use a main stage that will wrap your whole page just for more HTML semantic practice.

### Please make all the required changes before you submit another code review.
## Happy coding :tophat:

**Comment by danielufeli** [comment]
Thank you for your observation we have fixed all issues raised.

**Review by voscarmv — APPROVED** [review]
# Status: Project Approved ✨ 😎👍 💯 ✔️  

- Impressive use of animation on the product row.
- Nice use of CSS selectors for styling the form.

**[OPTIONAL]**
- Use a `font-family` and text size more similar to the original page.

### You may merge this branch and move forward ⛙🛣
### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**